pontoon/checks/utils/compare.py

```diff
@@ -0,0 +1,183 @@
+from collections import (
+    namedtuple,
+)
+
+from compare_locales.checks import getChecker
+from compare_locales.parser import (
+    FluentParser,
+)
+
+CommentEntity = namedtuple(
+    'Comment', (
+        'all',
+    )
+)
+
+File = namedtuple(
+    'File', (
+        'file',
+        'locale',
+    )
+)
+
+# Because we can't pass the context to all entities passed to compare locales,
+# we have to create our equivalents of compare-locale's internal classes.
+ComparePropertiesEntity = namedtuple(
+    'ComparePropertiesEntity',
+    (
+        'key',
+        'val',
+
+        # We'll remove these fields at some point, currently they're required because of the current
+        # implementation property files in compare-locales.
```

Comment:
I don't think that's how this will pan out. `raw_val` and `val` have different semantics, and for some checks I need one or the other.